### ISSUE TITLE
Issue culled by moderators

### ISSUE THREAD

**Issue by Robby262** [LibreTime/libretime#200]
Since we don't have a forum..
WE MADE A FORUM FOR LIBRETIME - IT WORKS! (been up for a month) - moderated out of existence in this Git!

We did provide a forum as a contribution to the project and BEFORE Haremare suggested making one, which was a reaction to our suggestion. Check the Datestamps. Ours is operational. I wrote it. Where is the Libretime one? I will happily redirect all traffic to it? How would cutting the only link to this site, that he paid for, benefit him? Here is the link that got 'moderated' and lit the fuse..

JohnnyC1951 commented on 11 Apr
[Deleted]

This HAD said: http://web-radio-support.websitesinternational.net/

JC was not best pleased about it and other posts that got 'moderated' without showing that they had been. Outraged would not cover it.

JohnnyC was not banned

Lie! Proof Below..

@Robbt - A reminder and just for info:

@hairmare commented on 11 Apr • edited
@LibreTime/maintainers These fruitless discussions have lead to me blocking the offending party at the orga level.
I also was copied on the email exchanges which are not reflected in the Git. I could post them here - but it got very childish all around. He had a back-channel into the AT Devs and much of the 2.51 & 2.52 fixes were his and merged in by Albert S.

We did provide a forum as a contribution to the project and BEFORE Haremare suggested making one, which was a reaction to our suggestion. Check the Datestamps. Ours is operational. I wrote it. Where is the Libretime one?
I will happily redirect all traffic to it and save our hosting fees. Please supply the URL. If you do not have one, I don't think we should believe you.

How would cutting the only link to this site, that he paid cash for, benefit him?

The full madness is here:
https://github.com/LibreTime/libretime/issues/156

I think that is team 'Websites International' and our family of stations done here, at least for now. JC may reverse this. If and when we see some maturity, structure, quality control, accountability.. who knows.
I will pass on your good wishes. Both JC and his wife are both being treated for cancer. I am sure we all wish them well.
I think it all comes down to Haremare. The sooner that he realizes that he is just a cog in the wheel and not the whole wheel - the better.

**Comment by Robby262** [comment]
So  Lucas. Want to play nice and fair? or want to take it to the web? Your choice? I see you already deleted a previous copy of this.. your call - i have a huge arsnenal - your call.. Let play commence..

**Comment by hairmare** [comment]
@Robby262 I consider your current behaviour to be harmful to the community and am asking you to self-correct this. Please stop attacking everyone that tries to help with your issues. We are trying to reach consensus on raised issues so we can consider implications to other users of the software/community, your way of adding to the discussion makes this harder than it should be.

You have repeatedly shown that you are not willing to follow both the [Code of Conduct](https://github.com/LibreTime/code-of-conduct/blob/master/CODE_OF_CONDUCT.md) nor the [C4 process](https://rfc.zeromq.org/spec:42/C4/) we are trying to utilize. We consider both the CoC and the C4 instrumental to the ongoing development efforts. After #199 I am past the point of being willing to explain this again, here is part of the C4 for your reference:

> 1. Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
> 2. To request changes, a user SHOULD log an issue on the project Platform issue tracker.
> 3. The user or Contributor SHOULD write the issue by describing the problem they face or observe.
> 4. The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
> 5. Users SHALL NOT log feature requests, ideas, suggestions, or any solutions to problems that are not explicitly documented and provable.
> 6. Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
> 7. To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
> 8. To submit a patch, a Contributor SHALL create a Platform pull request back to the project.

I did block JohnnyC at some point, I also unblocked him after a couple of weeks. I never removed the link he posted to your forum nor did I delete more than a snide reference he made to a member of legacy upstreams community. Please stop accusing people of stuff that the GitHub audit log clearly shows to be otherwise.

After #199 and this issue I am fully prepared to block members of the websites international team should they not make a clear effort to follow process and/or continue attacking our community members.

I'm ending this with another excerpt of the C4 to give some context on why I don't see any other way to proceed at this point.

> Administrators SHOULD block or ban "bad actors" who cause stress and pain to others in the project. This should be done after public discussion, with a chance for all parties to speak. A bad actor is someone who repeatedly ignores the rules and culture of the project, who is needlessly argumentative or hostile, or who is offensive, and who is unable to self-correct their behavior when asked to do so by others.

tldr: play nice or get blocked, no more drama please
